Library/Homebrew/dev-cmd/linkage.rb

```diff
@@ -22,6 +22,11 @@ def linkage
       if ARGV.include?("--test")
         result.display_test_output
         Homebrew.failed = true if result.broken_dylibs?
+        if OS.linux?
+          host_whitelist = %w[libc.so.6 libm.so.6 libgcc_s.so.1 libstdc++.so.6]
```

Comment:
Yes, all the `glibc` and `gcc` libraries should be whitelisted. There's an awful lot of them. `glibc` has 45 and `gcc` has 19. I wonder if there's a better way than listing them all explicitly.